roles/php/tasks/install-RedHat.yml

```diff
@@ -1,19 +1,39 @@
 ---
 
+- name: Ensure Repositories are installed (RedHat)
+  yum:
+    name: "{{ item }}"
+    state: present
+  loop: "{{ php_yum_repositories }}"
+  when: php_alt_repo and php_yum_repositories is defined and php_yum_repositories|length
+  become: true
+
+- name: Import remi GPG key.
+  rpm_key:
+    key: "{{ item }}"
+    state: present
+  loop: "{{ php_yum_keys }}"
+  when: php_alt_repo and php_yum_repositories is defined and php_yum_repositories|length
+  become: true
+
 - name: Ensure PHP packages are installed (RedHat)
   yum:
-    name: "{{ php_packages }}"
+    enablerepo: "remi,remi-safe,remi-php{{ php_version }}"
```

Comment:
isn't remi required only when php_alt_repo is true?

Reply:
Done